evalml/automl/automl_search.py

```diff
@@ -446,7 +446,7 @@ def __init__(
         allow_long_running_models=False,
         _ensembling_split_size=0.2,
         _pipelines_per_batch=5,
-        _automl_algorithm="iterative",
+        _automl_algorithm="default",
```

Comment:
The actual swap is here, I'm considering opening the `_automl_algorithm` parameter to be public as well.